src/app/public/modules/definition-list/definition-list.component.ts

```diff
@@ -15,12 +18,20 @@ import { SkyDefinitionListService } from './definition-list.service';
 })
 export class SkyDefinitionListComponent {
 
-  // TODO: figure out how to manage state here.
+/**
+ * Specifies the width of the label portion of the definition list.
+ * @default 90px
+ */
   @Input()
   public set labelWidth(value: string) {
     this.service.labelWidth.next(value);
   }
 
+/**
+ * Specifies the text to display in the value portion of the definition
+ * list when no value is available in a label-value pair.
+ * @default None found
```

Comment:
Steve told me that default values for `string` properties should include single quote marks.
```suggestion
 * @default 'None found'
```